pkg/resiliency/endpoints/bpf_backer.go

```diff
@@ -6,8 +6,9 @@ package endpoints
 import (
 	"fmt"
 
-	"github.com/cilium/cilium/pkg/endpoint"
 	"github.com/vishvananda/netlink"
+
+	"github.com/cilium/cilium/pkg/endpoint"
```

Comment:
nit: squashing these changes into the the previous commit would clean this diff up a bit